ftp-upload/cloudformation.yaml

```diff
@@ -88,4 +95,25 @@ Resources:
   SourceBucket:
     Type: AWS::S3::Bucket
     Properties:
-      BucketName: !Ref SourceBucketName
\ No newline at end of file
+      BucketName: !Ref SourceBucketName
+  EmptyCSVAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmName:
+        !Sub nla-pageview-${Stage}-alarm-empty-csv
+      AlarmDescription: Alert on receiving a small CSV file from Athena
+      Namespace: AWS/Lambda
+      MetricName: SizeOfCSV
+      Dimensions:
+        - Name: FunctionName
+          Value:
+            !Ref Lambda
+      Statistic: Sum
+      ComparisonOperator: LessThanThreshold
+      Threshold: 100
+      Unit: Megabytes
```

Comment:
Blimey, these must be pretty big files usually then?

Reply:
yeah around 170–180, here I just wanted to be conservative